proposals/nnnn-mt-registration-limitation.md

```diff
@@ -1,133 +1,122 @@
 
 
 ## Potential downsides
 
-* App registration sequence, especially the retry logic in Proxy, gets more complicated.
-* Since smartDeviceLink.ini file is not visible to mobile application developers, they may be confused that some apps are allowed to use a transport (such as Wi-Fi) but others aren't. They may also be confused that an app may be kept in HMI level NONE under a condition that is not open to them.
+* The resumption sequence in Core will be more complicated.
+* Since smartDeviceLink.ini file is not visible to mobile application developers, they may be confused that apps with certain AppHMITypes are launched automatically while other apps aren't. Or, they may also be confused that an app is automatically launched on a head unit from an OEM while it isn't on a head unit from another OEM.
 
 
 ## Impact on existing code
 
-This proposal impacts the basic app registration sequence in Core.
+* This proposal impacts resumption sequence in Core that runs after app's registration.
+* This proposal will not affect Proxy, HMI and RPC.
+
+
+## Out of scope of this proposal
+
+When a mobile projection app is connected through Bluetooth and Wi-Fi transports then the Wi-Fi transport is disconnected, HMI may want to terminate the app instead of showing a frozen screen. Since this proposal only changes the behavior of resumption, such use-case is not covered. OEMs may implement logic in HMI to terminate the app (using `BC.OnExitApplication` with `reason` being `USER_EXIT`) when the Secondary Transport is disconnected.
 
 
 ## Alternatives considered
 
-* Instead of adding the features in Core, update iOS Proxy to meet the basic requirement. That is, iOS Proxy always tries iAP connection before starting Wi-Fi transport (unless the app specifies to use TCP transport for debugging). When the Proxy uses both iAP and Wi-Fi transports, and iAP gets disconnected, Proxy closes Wi-Fi transport after certain time.<br>
-This approach should have less impact to the system since we do not update Core implementation. The disadvantage is that it is not a generic solution and is not flexible. Also, OEMs cannot choose to enable or disable the feature.
-* Instead of adding logic in Core, add it in HMI. The issue with this approach is that HMI cannot control app launching completely, since Core may launch an app using resumption.
-* Instead of rejecting RegisterAppInterface, send Start Service NACK during Version Negotiation. In this approach, Proxy's implementation can get more complex since it should initiate Start Service again after RegisterAppInterface is succeeded and another transport is established on it.
+* Instead of adding logic in Core, add it in HMI. The issue with this approach is that an app will be shown on HMI for a short time until HMI terminates it. Such behavior may lead to user's another confusion. Also, the author thinks that logic to manage and control apps' states should be implemented in Core, not in HMI.

```

Comment:
Please update the third sentence to "Such behavior may lead to additional user confusion."